src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -155,6 +154,30 @@ public void runningBuild() throws Exception {
         d.assertItemNames(5);
     }
 
+    @Test
+    public void notAddChildren() throws Exception {
+        JenkinsRule.WebClient client = r.createWebClient();
+        SampleComputedFolder s = r.jenkins.createProject(SampleComputedFolder.class, "s");
+
+        assertThat(client.getPage(s).getByXPath("//a[contains(text(), \"New Item\")]").size(), is(0));
+
+        s.kids.add("A");
+        s.recompute(Result.SUCCESS);
+
+        assertThat(client.getPage(s).getByXPath("//a[contains(text(), \"New Item\")]").size(), is(0));
+    }
+
+    @Test
+    public void runByTrigger() throws Exception {
+        SampleComputedFolder s = r.jenkins.createProject(SampleComputedFolder.class, "s");
+        s.assertItemNames(0);
+
+        s.addTrigger(new PeriodicFolderTrigger("1m"));
+
+        Thread.sleep(65000);
```

Comment:
Yes, but that doesn't verify that FolderCron is actually checking for triggers and calling them. Or do you think to verify that is not worth waiting this extra time for tests to finish?